### PR TITLE
remove maints access from passengers

### PR DESCRIPTION
**Pull request by MilonPL**
## About the PR

title

## Why / Balance

it made sense back when passengers were you know... assistants... but right now they aren't even crew anymore according to the SOP
it makes no sense for them to have any extra access at all, especially critical infrastructure like maints
yes tiding and maints loot is fun (as long as you're the one tiding) but not :sparkles: *MRP* :sparkles: and at best annoying for everyone else 
and there's a lot of service works that are basically "greytider plus" anyways

## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- tweak: Passengers no longer have maintenance access.

### CODE REVIEW

**Comment by dotCatshark** [comment]
finally. even less fun things for passengers and assorted roles to do.

**Comment by SolStar2** [comment]
This seems sensible and good for the direction of delta-v

**Comment by SolStar2** [comment]
If you want to do tidey shenanigans a service worker makes a lot more sense than a passenger anyways and may get passengers to try to work more with crew instead of disappearing in maints half the shift.

**Comment by Avalon-Proto** [comment]
Uh, isn't half the point of Maints to be like a fire escape/emergency exit too? I understand you don't like tiders too much but isn't that access kinda needed so that they don't get stuck in a trait fire etc

**Comment by MilonPL** [comment]
If you're stuck in a trit or a plasma fire, maints won't save you. And if you don't have a holofan, you'll just superheat half of the station anyways.

**Comment by Avalon-Proto** [comment]
How about our passenger antags then. Losing Maints access is a massive blow as now you literally cannot access the most common hiding spot for antags. Better give em like 5-10 extra TC to compensate for the fact they can't run and hide anymore

**Comment by MilonPL** [comment]
> How about our passenger antags then. Losing Maints access is a massive blow as now you literally cannot access the most common hiding spot for antags. Better give em like 5-10 extra TC to compensate for the fact they can't run and hide anymore

airlock access override, jaws of life, stealing IDs, *getting a job so that you have more than just maints access*, microwaving IDs
you can get pretty creative

in the end you're *choosing* to play passenger and be an antag, you know that this job doesn't have any access

**Comment by Avalon-Proto** [comment]
It's still a nerf to the baseline, and now takes the most work out of all antags to get a basic access. Should get a small TC buff since this PR Essentially says "If a passenger is anywhere but public areas it's suspicious"

**Comment by BlitzTheSquishy** [comment]
Sorry milon but this is an L PR, please try harder next time

**Comment by therealDLondon** [comment]
Not even the HRP Cosmatic Drift does this, the maints are there as hidng spots for antags, alternate routes for shortcuts, fire escapes and more.
This PR would take away part of the core heart/soul of what Space Station is, and without a consise view from the direction team, this PR feels off to me.
Sorry, I cant approve this

**Comment by MilonPL** [comment]
> Not even the HRP Cosmatic Drift does this

Our culture is drastically different than CD's, and iirc they do not allow maints hunting anyways, so obviously it's not an issue. 
I do not think that a passenger just "tiding" is engaging in any meaningful way. It's not really telling a story, it doesn't engage other players, there's no collaborative aspect. You can use maints as a space to build your own things by playing engineering. And all *crew* can access the maints anyways, it's just *passengers* that can't. People who are supposed to be *visitors*. If you're a passenger traitor and desperately need access, there's plenty of ways to get it.

**Comment by Avalon-Proto** [comment]
You say it doesn't make a story but I can give you a big example of a passenger who does make a story out of it. Da Big Cheese emporium. They start the shift gathering junk and items from Maints and set up a trade post with all those items inside. Working off barters to get more and better goods

**Comment by Radezolid** [comment]
Against this, PR, it would only make passengers valid the moment they aren't in a public area, it would destroy friends gathering spots that some users like to do and it just feels wrong. This sounds more like something that should have be done in a poll on discord (seeing we got a channel for that)

**Comment by Lyndomen** [comment]
Passengers can hack in if they really wanted to. They didn't have maints access a year ago, and it was fine. If you don't want to play a role that has limited your access, there are 90 others

**Comment by Lyndomen** [comment]
> You say it doesn't make a story but I can give you a big example of a passenger who does make a story out of it. Da Big Cheese emporium. They start the shift gathering junk and items from Maints and set up a trade post with all those items inside. Working off barters to get more and better goods

Ok so they can hack in if they want to do black market shit, or ask HOP for maints access

**Comment by Lyndomen** [comment]
> Against this, PR, it would only make passengers valid the moment they aren't in a public area, it would destroy friends gathering spots that some users like to do and it just feels wrong. This sounds more like something that should have be done in a poll on discord (seeing we got a channel for that)

Valid is a concept we really want to move away from. Passengers are a challenge to some extent, you have none of the privileges and all the suspicion

**Comment by Lyndomen** [comment]
> Not even the HRP Cosmatic Drift does this, the maints are there as hidng spots for antags, alternate routes for shortcuts, fire escapes and more. This PR would take away part of the core heart/soul of what Space Station is, and without a consise view from the direction team, this PR feels off to me. Sorry, I cant approve this

Core part of the soul that didn't exist a year ago? Idk, this isn't removing tiding it's making it more difficult

**Comment by Lyndomen** [comment]
> Sorry milon but this is an L PR, please try harder next time

Make an issue :godo:

**Comment by Lyndomen** [comment]
> How about our passenger antags then. Losing Maints access is a massive blow as now you literally cannot access the most common hiding spot for antags. Better give em like 5-10 extra TC to compensate for the fact they can't run and hide anymore

Passengers opt in for the extra difficulty. Play service worker or another role if you want optimal gamer role

**Comment by dotCatshark** [comment]
> Ok so they can hack in if they want to do black market shit, or ask HOP for maints access

So it changes nothing, and passengers just go ask HoP for maints access before tiding. What's the point of this PR again? Well, according to what's on the tin, to make it so that passengers can't go into maints, or at least so that its more difficult. So if they'll ask for access anyways (and it's likely most HoPs will just say yes, frankly), what's the point? And how is hacking in any LESS low roleplay than just, walking in there with access and doing it ANYWAYS?

**Comment by MilonPL** [comment]
> So it changes nothing, and passengers just go ask HoP for maints access before tiding. What's the point of this PR again? Well, according to what's on the tin, to make it so that passengers can't go into maints, or at least so that its more difficult. So if they'll ask for access anyways (and it's likely most HoPs will just say yes, frankly), what's the point? And how is hacking in any LESS low roleplay than just, walking in there with access and doing it ANYWAYS?

I think that even if they have to ask for access, that's still better. At least there is *some* sort of interaction. Ultimately up to the players, but that's an opportunity for them to roleplay.

**Comment by Lyndomen** [comment]
> > Ok so they can hack in if they want to do black market shit, or ask HOP for maints access
> 
> So it changes nothing, and passengers just go ask HoP for maints access before tiding. What's the point of this PR again? Well, according to what's on the tin, to make it so that passengers can't go into maints, or at least so that its more difficult. So if they'll ask for access anyways (and it's likely most HoPs will just say yes, frankly), what's the point? And how is hacking in any LESS low roleplay than just, walking in there with access and doing it ANYWAYS?

Because RP is more than just talking. Seeing as passenger fry themselves on a door is a CLASSIC I have missed.

**Comment by dotCatshark** [comment]
> Passengers opt in for the extra difficulty. Play service worker or another role if you want optimal gamer role

Passengers don't opt in for the extra difficulty. Passengers play passenger because it offers freedom and a total lack of attachment to anything. And also because again, it's an overflow role with infinite slots.

**Comment by Lyndomen** [comment]
Also almost forgot, gotta down vote this now cause it should be an issue :godo: report me all you want MiLon

**Comment by MilonPL** [comment]
> Also almost forgot, gotta down vote this now cause it should be an issue :godo: report me all you want MiLon

no because it's a complete review-ready PR
if it's get closed it's whatever :blunt:

**Comment by Lyndomen** [comment]
> > Passengers opt in for the extra difficulty. Play service worker or another role if you want optimal gamer role
> 
> Passengers don't opt in for the extra difficulty. Passengers play passenger because it offers freedom and a total lack of attachment to anything. And also because again, it's an overflow role with infinite slots.

Lack of attachment means u don't need maints, u ain't maintaining anything. If you want no attachment, turn antag off and no worries. Worst case scenario, ask any head for access and why

**Comment by dotCatshark** [comment]
> > > Ok so they can hack in if they want to do black market shit, or ask HOP for maints access
> > 
> > 
> > So it changes nothing, and passengers just go ask HoP for maints access before tiding. What's the point of this PR again? Well, according to what's on the tin, to make it so that passengers can't go into maints, or at least so that its more difficult. So if they'll ask for access anyways (and it's likely most HoPs will just say yes, frankly), what's the point? And how is hacking in any LESS low roleplay than just, walking in there with access and doing it ANYWAYS?
> 
> Because RP is more than just talking. Seeing as passenger fry themselves on a door is a CLASSIC I have missed.

Seeing a passenger fry themselves on a door isnt roleplay, if that's what you're saying; it's not an interaction, it's not anything. It's just you seeing a passenger try and hack into somewhere and thinking "haha funny" (which it is) but it's not somehow conducive to a better roleplay environment.

**Comment by boogiebogus** [comment]
> airlock access override, jaws of life, stealing IDs, _getting a job so that you have more than just maints access_, microwaving IDs you can get pretty creative
> 
> in the end you're _choosing_ to play passenger and be an antag, you know that this job doesn't have any access

technical assistant slot stocks booming

**Comment by dotCatshark** [comment]
> > > Passengers opt in for the extra difficulty. Play service worker or another role if you want optimal gamer role
> > 
> > 
> > Passengers don't opt in for the extra difficulty. Passengers play passenger because it offers freedom and a total lack of attachment to anything. And also because again, it's an overflow role with infinite slots.
> 
> Lack of attachment means u don't need maints, u ain't maintaining anything. If you want no attachment, turn antag off and no worries. Worst case scenario, ask any head for access and why

Okay, so is the argument is that they don't maintain anything so they don't need the access? If that IS the argument, why does scientist, chaplain, doctor, paramed, and every single other role aside from engineering have maintenance access.

**Comment by Lyndomen** [comment]
Because all of those do maintain the station in their own capacity :godo:

**Comment by Lyndomen** [comment]
and for everyone saying that "tiding is an IC issue", this is the IC issue in question. This gives Sec authority to actually arrest people for B&E, which gives them something to do- something everyone loves to throw around.

**Comment by dotCatshark** [comment]
> Because all of those do maintain the station in their own capacity :godo:

And why would, say, a chef, or a chaplain, or a cargo technician, or a secoff, need maintenance access? Yes, they "maintain" the station as in "they have a role in the functioning of the station" but unless you were making a very weird analogy where you didn't mean maintaining as in, literally going into maints and fixing shit, then they have absolutely ZERO role in station maintenance.

**Comment by IamVelcroboy** [comment]
I remember this was tried on Nyano and reverted like a week later. You're going to force tiders into public areas to do their tider things instead of giving them an outlet to do those things in an area of the station that's not crucial to most people's jobs which will increase conflict/disruption for players with a station role.

**Comment by MilonPL** [comment]
> I remember this was tried on Nyano and reverted like a week later. You're going to force tiders into public areas to do their tider things instead of giving them an outlet to do those things in an area of the station that's not crucial to most people's jobs which will increase conflict/disruption for players with a station role.

Respectfully, this isn't Nyano, and a lot has changed since then. If it sucks, the revert button is always there.

**Comment by IamVelcroboy** [comment]
Nyano or not, my main point still stands

**Comment by boogiebogus** [comment]
> Lack of attachment means u don't need maints, u ain't maintaining anything. If you want no attachment, turn antag off and no worries. Worst case scenario, ask any head for access and why

For an actual serious answer to this: all i see is people taking roles with an actual job (technical assistant cough cough) and immediately fucking off because nobody's going to notice their absence.

> Because RP is more than just talking. Seeing as passenger fry themselves on a door is a CLASSIC I have missed.
> and for everyone saying that "tiding is an IC issue", this is the IC issue in question. This gives Sec authority to actually arrest people for B&E, which gives them something to do- something everyone loves to throw around.

awww yeaa.... roundstart self antagging pass to b&e into maints

**Comment by HTMLSystem** [comment]
I don't like this or either. Whenever I play passanger and tide now a days, I fuck off in one of the maints secret rooms and build a web. And then I enjoy my web. And then people stumble upon my web and they go "holy shit, this is nice" and it's awesome. Perhaps when you're done you can leave a note with your characters thoughts and feelings and cryo, then later someone can come by and read them. Sometimes it's nice to go maints hunting and show off the cool shit you have like the fancy cowboy boots, a mosin, machete, those high heeled boots, a pair of insuls, or to gather all the strange pills and spaceshrooms before some other tider eats them all. 
I do genuinely think this would remove more potential rp then add it.

**Comment by HTMLSystem** [comment]
> For an actual serious answer to this: all i see is people taking roles with an actual job (technical assistant cough cough) and immediately fucking off because nobody's going to notice their absence.

And also this! If we remove passanger maints access there's gonna be a lot more tech assists and service workers tiding and not doing an actual job, thus creating more of a problem. Those are learner roles and shouldn't be hogged by people just seeking maints access and tiding.

**Comment by boogiebogus** [comment]
> I don't like this or either. Whenever I play passanger and tide now a days, I fuck off in one of the maints secret rooms and build a web. And then I enjoy my web. And then people stumble upon my web and they go "holy shit, this is nice" and it's awesome. 

adding on to this--maps are literally built to support maints diving. you shouldn't have to be an engineer to build and run a maints bar. you shouldn't need a job to enjoy the view on hive. you shouldn't need a job to stare into the singulo on edge.

**Comment by flowerfall-system** [comment]
Why??? What??? So just, forget all about Maints as a safety net, forget about hiding from threats or weaving through the station unseen, forget about being a CREATURE that likes being in the dark, forget about maints bars and hidey holes, forget about the time-honored tradition of tiding --- if you flush yourself down Dispo, are you just FUCKED? If Passengers don't need maints access because "they don't need it", don't forget to remove Maints access for:

-Service Worker, Reporter, Musician, Mime, Librarian, Clown, Chef, Boxer, Botanist, Bartender, HoP, Psychologist, Med Intern, Med Doctor, Chemist, Salvage, Couriers, Cargo Technicians, LO, Prosecutor, Attorney, Clerk, CJ, Scientist, RA, Mantis, Chaplain, and Mysta, too. Only Engi, Parameds (CMO by extension), Sec, and the Janitor need access to Maints, and the Janitor only because some Dispos can only be accessed through maints.

this kills WAAAAAAAAAAAAAAAY more stories than it makes, and to what end? so Passengers can form a line and ask for Maints access until the meta turns into "HoP denies all Maints access requests", because IT WILL, and then what? then what happens? what about the RP stories of the hidey holes, the maints goblins, the THINGS, the scary escapes -- because let's not forget Maints are a backdoor too! Let's not forget all the stuff that spawns there SPECIFICALLY so people can take it! Let's not forget about the EVA lockers, the gas cans, let's not forget all that!!

MASSIVE L PR, Milon, MASSIVE L. This is on par with, maybe even WORSE, than EmoGarbage's PR to replace Salv mini-jetpacks with fire extinguishers. On further thought, though -- yeah, it is worse! EmoGarbage's PR was inconvenient and silly and inconsistent with the worldbuilding -- this one actively SABOTAGES server culture as it is now. We're with HTML on this -- run this PR, and you'll see a LOT more roles being taken just for the sake of maints access. TERRIBLE idea, well and truly.

Hey, what about the ESCAPE PODS???????

**Comment by MilonPL** [comment]
Escape pods are a mapping issue, you *shouldn't* need to go find your way to them through maints. 
<details><summary>Tortuga:</summary>
<br>

image

</details>

**Comment by Lyndomen** [comment]
resolves https://github.com/DeltaV-Station/Delta-v/issues/2179
:gigabased:

**Comment by Lyndomen** [comment]
> > Lack of attachment means u don't need maints, u ain't maintaining anything. If you want no attachment, turn antag off and no worries. Worst case scenario, ask any head for access and why
> 
> For an actual serious answer to this: all i see is people taking roles with an actual job (technical assistant cough cough) and immediately fucking off because nobody's going to notice their absence.
> 
> > Because RP is more than just talking. Seeing as passenger fry themselves on a door is a CLASSIC I have missed.
> > and for everyone saying that "tiding is an IC issue", this is the IC issue in question. This gives Sec authority to actually arrest people for B&E, which gives them something to do- something everyone loves to throw around.
> 
> awww yeaa.... roundstart self antagging pass to b&e into maints

hacking a door into maints isnt self-antag, it isnt antagonistic :godo:
command members challenge: fire shitty crew members- IMPOSSIBLE

**Comment by Avalon-Proto** [comment]
So. A few more points for this
- Escape pods: On 90% of maps they are entirely inaccessible without Maints access, so if we want to take Maints access away we need to make sure all maps are ready for this
- Tiding: While tiders can be annoying. Overall they are generally harmless and can actually be helpful as well
- ToolRooms: You say passengers can just use a you tool but only around 3 maps in our rotation have them publicly accessible, meaning that it's nigh impossible for tiders to get budgets to fry themselves on doors like you do want lyndo
- Emergency closets: All locked behind Maints access doors besides a few spots. So you want the passenger who needs an emergency EVA cause they are a they are in suddenly spaced to die in the hall banging on the Maints door with an emergency suit locked behind it
- Job roles: so say you lock Maints access off to passengers but now people who just want to do their own thing in round have to play service workers,RAs, TAs and interns to get their Maints access. This takes these roles away from players who want to learn how to play the game unless you give them a maximum time allowed requirement, which we specifically don't do so that players can make characters who fit in those learner roles. Instead our players who just want to do their own thing will take this roles and vanish into Maints never to be seen again
I'm sorry but as it stands this PR hurts the server more than helps it, and I think it should be closed.

**Comment by dvir001** [comment]
@MilonPL Speaking of, why don't we actually just add assistants, I dont even get the point of not having assistants to start with.

Isn't there support to allow multi PDA options renaming? can it also support adding maint to people who pick assistants? can assistant be a timed lock to 3 hours of passenger?

🤷

**Comment by MilonPL** [comment]
All doable yes, I'm actually in favor of just nuking passengers and tourists/visitors. Off-Duty crew makes sense.

**Comment by IamVelcroboy** [comment]
Someone make Casino Station for the tourists

**Comment by dvir001** [comment]
> Someone make Casino Station for the tourists

What is the library for?
@MilonPL Port the playing cards code from frontier.

**Comment by Colin-Tel** [comment]
I could go either way on this, tbh. I'll just say that it would've been nice if another solution (such as the alternate PDA) was included in the "remove access" PR.
I'm gonna poll the public on this.

**Comment by MilonPL** [comment]
> I could go either way on this, tbh. I'll just say that it would've been nice if another solution (such as the alternate PDA) was included in the "remove access" PR. I'm gonna poll the public on this.

I like the assistant PDA solution honestly. But do we just nuke passengers then? Not sure.

**Comment by FieldCommand** [comment]
Not much to be said, as it has been said before in other posts on this PR. Other than that, I think this will do more harm than good to delta-v community. More communication between delta-v management and the public is truly needed.

**Comment by Lyndomen** [comment]
for clarity, delta v management isnt the one really doing this. milon is a maintainer, but in this capacity just a contributor. 

similarly, I quit so my comments are just my own

**Comment by Colin-Tel** [comment]
> More communication between delta-v management and the public is truly needed.

What would you recommend beyond polling the public and waiting for their input?

**Comment by LunaLettiere** [comment]
@Lyndomen 
> Passengers opt in for the extra difficulty. Play service worker or another role if you want optimal gamer role

Hard untrue, you _start_ as a passenger, just dying to a tarantula or something because you find yourself in a compromising situation and can't leave is only going to deter players.

@Avalon-Proto 
> * Escape pods: On 90% of maps they are entirely inaccessible without Maints access, so if we want to take Maints access away we need to make sure all maps are ready for this
> * Tiding: While tiders can be annoying. Overall they are generally harmless and can actually be helpful as well
> * ToolRooms: ...
> * Emergency closets: All locked behind Maints access doors besides a few spots...
> * Job roles: ... want to do their own thing in round have to play service workers, RAs, TAs and interns to get their Maints access. This takes these roles away... our players who just want to do their own thing will take this roles and vanish into Maints never to be seen again
>   I'm sorry but as it stands this PR hurts the server more than helps it, and I think it should be closed.

Hard agree on all points. Lets not forget all the fun fluff that's in all of the maps' maintenance, silly bars, little dojos, fun items, etc. The game would just become more boring. The poll appears to be getting destroyed with _"NO"_ too, the community is against this change for so so many reasons, Big L PR. People aren't going to reconfigure the maps and the maps already have **actual** _maintenance_ areas in the form of all the little engineer access points to substations and external areas and such. This just serves to make the game more difficult for antags and less friendly to new people and exploration. 

Besides, the answer to this problem being "Just Hack Into Them" is _way_ more LRP and will be introduced as self-antag behaviour in a future rule update once this is implemented. If you don't have a reason to hack into epi/engi/med/sec/whatever without being an antag, why would you have a reason to hack into _maints_ if you wanted to spelunk a little bit.

This kind of shit is just actively detrimental to gameplay for the purpose of "Muh RP" without taking into consideration the ripple effect PR's like this have.

**Comment by Tegotia** [comment]
Up front I'll say my perspective is of someone with very limited time on Delta-V, who hasn't played SS14 in general for several months, and of course this is all opinion and I don't think mine is more valuable than anyone else's. But all that said, I think this is a _very_ bad idea.

The core gameplay loop of SS14 is based around _disruption_. Everyone has roles, yes, but the point of the game is not, primarily, about playing out those roles to their fullest. It is about _things going wrong_. The balance here is tenuous - there needs to be some promise that it is possible to achieve your goals or people are going to get frustrated and stop trying, but on the other hand if there is no chance of anything going wrong at all, the game simply becomes a task simulator with no real conflict or stakes.

There is an issue though with the way a lot of antagonists work in that the chaos they cause is limited in scope. A traitor, for example, is most of the time only going to cause serious disruption to the people / departments that their objectives target, assuming they even manage to achieve anything at all. This leaves a lot of people who aren't going to be involved in that gameplay loop at all just playing the task simulator game. I do think Delta-V handles this better than some other servers since I've noticed it has higher rates of random events & minor antags, but in a social game there's only so much you can do with PvE content without it feeling like another chore.

This is where passengers come in. Being roles with effectively no responsibility, they are more free to add some dynamism to the round through whatever antics they get up to. But at the same time, since they are still crewmembers (skipping over passenger antags here because the role is really secondary in this case) they are limited in what kind of things they can do without consequences. They can cause a little bit of disruption to shake up events, but ultimately they probably aren't going to totally derail someone's round. I think this makes them _super_ important as a role, not because of what their "in-character" purpose is, but the role they fill to the overall gameplay loop.

That's why removing maints access for passengers is particularly bad - it takes away effectively the one purpose that role has, and that is to be effectively little gremlins that aren't going to ruin's anyone's day but can shake things up and break up the monotony of just doing your role's tasks. Technically, yes, it will still be _possible_ to do that without maints access, but frankly it would make it a whole lot less fun and effective. As it is they also fill this role better than random events too - being a social game, it's crucial that there's a human element to these things to keep things feeling alive. But I suspect that's also the reason that people want to remove maints access from them.

Some people do take the antics too far, and I imagine just about everyone has a memory of someone disrupting the round and ruining their day when they weren't even an antag, and that feeling definitely sucks. I do think some people get too into their roles and forget that the point of the game is that they _will_ get disrupted and will not likely be able to just do their things without something going wrong, but I say that not to discredit the frustration but just to acknowledge that there is, again, a tenuous balance here. 

I think there is more work that needs to be done to find the right amount of chaos, but wholesale removing gameplay from a role is way too scorched earth.

This ended up being way longer than I thought it would be so hopefully if anyone did actually read this all you don't feel like it was a waste of time :)

**Comment by FieldCommand** [comment]
> > More communication between delta-v management and the public is truly needed.
> 
> What would you recommend beyond polling the public and waiting for their input?

A few things: A mission statement and vision of Delta-V that is easy to find for everyone. Some sort of blog that is updated semi-regularly and also easy to find that is about progress, updated, and other things that is to drum up support and convey the future of Delta-V. I don't know if they still happen, but the monthly voice call hangout with the managers of Delta-V should have a written report of the highlights about what was said in said call. Not everyone can attend them.

Discord is all wall and good, but if the information is hard to find and/or gets drowend out, it becomes meningless. It also needs to be clear and easy to read. 
Basically, make a few dedicated channels (much like "Rules" and "Information") on Discord that do what I said above.
And in general, just more communication about things and statuses in general about Delta-V.

**Comment by dvir001** [comment]
I honestly think the core issue here that passengers are not passengers and have access to what in a logical way should be the core maintenance areas of a space station, off hands from a passenger.

And if I'm being honest, I think a first step will be to have another option for a PDA, or even another role to replace or to add that is an actual assistant by name.

I feel this is more a culture shift PR then an ID access issue.

If you think about it, the passenger role can still stay as an actual passenger, while a new role "assistant" can have the maintenance access to it, and more RP options at hand to work for departments or to serve as a connection from one to another while still having no real boss (HOP boss, minus service headset?).

If you ask me, don't remove passengers, do remove maintenance access from them.
Add the assistant job with a low playtime requirement that replace the old passenger.

**Comment by SolStar2** [comment]
I think making an assistant pda with those accesses would satisfy both sides of what I see people want.

**Comment by SolStar2** [comment]
Then you get a choice and a more consistent accesses with what you expect a passenger to have which means they would need to approach things in a way that encourages what I think passengers should be. It puts a layer of interaction with so many ways to approach that can reflect on how your character would behave.

**Comment by Zavender9** [comment]
Not a total L, I think you're onto _something_ but this isn't really the way.

**Comment by deltanedas** [comment]
would be better separating assistant (and off duty crew) and passenger (visitor/student/tourist) so if people want to rp someone that doesnt have ties with nt they can
